### ISSUE TITLE
DrawerNavigator no longer unmounts/remounts components on navigation

### ISSUE THREAD

**Issue by JeremyBradshaw7** [react-navigation/react-navigation#6080]
I've just upgraded from react-navigation v1 through v2 to v3. I've just noticed that components I navigate to from DrawerNavigator don't unmount and remount on choosing different options from the menu. Was this documented somewhere in the upgrade path? I didn't notice it. And what's the workaround?

**Comment by JeremyBradshaw7** [comment]
Sorry, I did find it in the 3.0 post, only problem is the new unmountInactiveRoutes config parameter is throwing an error:

```
src/Routes.tsx:417:5 - error TS2345: Argument of type '{ contentComponent: any; unmountInactiveRoutes: boolean; drawerPosition: "left"; contentOptions: { activeBackgroundColor: string; activeTintColor: string; inactiveTintColor: string; }; }' is not assignable to parameter of type 'DrawerNavigatorConfig'.
  Object literal may only specify known properties, and 'unmountInactiveRoutes' does not exist in type 'DrawerNavigatorConfig'.

417     unmountInactiveRoutes: true,
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by JeremyBradshaw7** [comment]
To get round it I've added this line to the typescript definition file:
![image](https://user-images.githubusercontent.com/36155348/61144385-aaf55800-a4cc-11e9-8260-58f94617ee6c.png)

**Comment by brentvatne** [comment]
added type and released in 3.11.1, thanks

**Comment by JeremyBradshaw7** [comment]
Thanks for turning this around so quickly @brentvatne
